packages/feedback/src/environments/environment.ts

```diff
@@ -0,0 +1,16 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=local` then `environment.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: false,
+  name: 'local',
+  api: 'https://localhost:8080',
+  graphqlAPI: 'https://localhost:8080/graphql',
+  client: 'https://localhost:4200',
+  authIssuer: 'https://auth.stage.redhat.com/auth',
+  matomoSiteID: null, // Site ID from Piwik Analytics
+  serviceNowURL: 'https://redhatqa.service-now.com',
+  subscription: 'wss://localhost:8080/subscriptions'
+};
```

Comment:
Can you please remove this hosts from here and please use the .env files for managing this information

Reply:
Done.